main.go

```diff
@@ -12,17 +12,38 @@ import (
 	"reflect"
 )
 
-var socket = flag.String("socket", "", "Socket to listen into")
+var dump = flag.String("dump", "", "Dump info about `plugin` as a MessagePack object")
```

Comment:
Minor: could we give it a more specific name? e.g., `dump-plugin-schema` or some such

Reply:
sure, why not...